specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,3027 @@
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "JobMetadata": {
+      "properties": {
+        "displayName": {
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "jobId": {
+          "format": "uuid",
+          "type": "string"
+        },
+        "lastUpdateDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "status": {
+          "enum": [
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed",
+            "cancelled",
+            "cancelling"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": false,
+            "name": "State"
+          }
+        }
+      },
+      "required": [
+        "jobId",
+        "lastUpdateDateTime",
+        "createdDateTime",
+        "status"
+      ],
+      "type": "object"
+    },
+    "JobState": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobMetadata"
+        },
+        {
+          "$ref": "#/definitions/JobErrors"
+        },
+        {
+          "$ref": "#/definitions/Pagination"
+        }
+      ]
+    },
+    "JobErrors": {
+      "properties": {
+        "errors": {
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "AnalyzeTextJobStatistics": {
+      "properties": {
+        "statistics": {
+          "$ref": "#/definitions/RequestStatistics"
+        }
+      },
+      "type": "object"
+    },
+    "TasksState": {
+      "properties": {
+        "tasks": {
+          "properties": {
+            "completed": {
+              "type": "integer"
+            },
+            "failed": {
+              "type": "integer"
+            },
+            "inProgress": {
+              "type": "integer"
+            },
+            "total": {
+              "type": "integer"
+            },
+            "items": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AnalyzeTextLROResult"
+              }
+            }
+          },
+          "required": [
+            "total",
+            "completed",
+            "failed",
+            "inProgress"
+          ],
+          "type": "object"
+        }
+      },
+      "required": [
+        "tasks"
+      ],
+      "type": "object"
+    },
+    "TaskState": {
+      "properties": {
+        "lastUpdateDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "status": {
+          "enum": [
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed",
+            "rejected",
```

Comment:
just wondering, what is the difference between "failed" and "rejected" for a user?

Reply:
This is a good question.  I'm actually not seeing any usage of this value in the service.  Let me follow up with a few folks on my side to ask about the rationale here.

Reply:
I recently heard back that there is no plan to support the `rejected` status, so I've removed it here.

It is included in every Swagger definition prior to this one though, so maybe this needs to have an issue created for it to address in previous API versions.